File: app/src/main/java/com/github/neowen/apibasedemo/ApiItem.java
package com.github.neowen.apibasedemo;

import android.app.Activity;

/**
 * Created by ZWX on 15/10/15.
 */
public class ApiItem {

    public final String mTitle;
    public final Class<? extends Activity> mClazz;

    public ApiItem(String mTitle, Class<? extends Activity> mClazz) {
        this.mTitle = mTitle;
        this.mClazz = mClazz;

    }
}

File: app/src/main/java/com/winson/apibasedemo/base/ApiItem.kt
package com.winson.apibasedemo.base

import android.app.Activity
import androidx.fragment.app.Fragment

/**
 *@date on 2019/1/14
 *@Author Winson
 */
class ApiItem(val title: String) {

    var activityClassName: Class<out Activity>? = null
    var isFragment = false
    var fragment: Fragment? = null

}